cypress/e2e/spec.cy.js

```diff
@@ -68,3 +68,22 @@ describe("Location", () => {
     });
   });
 });
+
+// ✅ --- PASSING ---
+describe("Checking the 404 page", () => {
```

Comment:
Great work on checking with a test that the page directs to a 404 once clicked on story time 🧪